test/cljam/io/vcf/util_test.clj

```diff
@@ -81,32 +81,37 @@
     "0/0" [[0 false] [0 false]]
     "0/1" [[0 false] [1 false]]
     "1/1" [[1 false] [1 false]]
+    "/0|1" [[0 false] [1 true]]
+    "|1/0" [[1 true] [0 false]]
     "./." [[nil false] [nil false]]
     "0|0" [[0 true] [0 true]]
     "0|1" [[0 true] [1 true]]
     "1|1" [[1 true] [1 true]]
     ".|." [[nil true] [nil true]]
     "0/1/2" [[0 false] [1 false] [2 false]]
-    "0/1|2" [[0 false] [1 false] [2 true]]))
+    "0/1|2" [[0 false] [1 false] [2 true]]
+    "|0/1/2" [[0 true] [1 false] [2 false]]))
 
 (deftest about-stringify-genotype
   (are [?gt ?expected]
        (= (vcf-util/stringify-genotype ?gt) ?expected)
     nil nil
     [[0 true]] "0"
     [[1 true]] "1"
-    [[0 false]] "0"
-    [[1 false]] "1"
```

Comment:
I dropped these tests for unphased haplotypes (which should be denoted as "/0" or "/1" using the explicit phasing indicator), as I'm not sure if those are valid notationally and conceptually.